applications/pelion_client/src/modules/oma_stopwatch.cpp

```diff
@@ -76,11 +90,30 @@ static void notification_status_callback(const M2MBase& object,
 {
 	LOG_DBG("Resource %p notification, object:%s status:%d", (void*)&object,
 		log_strdup(object.uri_path()), status);
-	if (status == M2MBase::MESSAGE_STATUS_DELIVERED) {
-		update_pending = false;
+	switch (status) {
+	case M2MBase::MESSAGE_STATUS_SENT:
+		/* This is ok - but wait for ACK */
+		break;
+	case M2MBase::MESSAGE_STATUS_BUILD_ERROR:
+	case M2MBase::MESSAGE_STATUS_RESEND_QUEUE_FULL:
+	case M2MBase::MESSAGE_STATUS_SEND_FAILED:
+	case M2MBase::MESSAGE_STATUS_REJECTED:
+		LOG_ERR("Error during %p resource update, object:%s status:%d",
```

Comment:
If this is ERR and possible can occur on CI, it may cause test fail.
@piwe-nordic any opinion?

I am adding approve as otherwise everything is ok. If Piotr says CI may complain then let's use WRN instead.

Reply:
IMHO Error should be used when the system cannot restore its functionality. If only one of the messages cannot be sent/resend it should be a warning. As for if Ci "might complain" I'm not versed in twister behavior regarding warnings and errors. If it is more robust to print WARN then that is my recommendation.

Reply:
Fixed